server/engine/src/main/java/org/eclipse/lsp/cobol/cli/command/CliAnalysis.java

```diff
@@ -70,6 +68,7 @@ public Integer call() throws Exception {
         cliClientProvider.setCpyExt(createCopybooksExtensions());
         JsonObject result = new JsonObject();
         result.addProperty("uri", src.toURI().toString());
+        result.addProperty("language", dialect.getId());
```

Comment:
Probably now you can remove it from line 85

Reply:
updated